src/redux/slices/example.js

```diff
@@ -5,18 +5,28 @@ export default function exampleResponse() {
         id: '550e8400-e29b-41d4-a716-446655440000',
         type: 'meal',
         attributes: {
-          time: 123345534,
+          childId: '5ee62461-adb8-4618-a110-06290a787223',
+          time: '12:54',
```

Comment:
Could you pls add a timestamp to `time` field? That way this example data is consistent with the API.

Reply:
We are not using this mock data anymore. The data being fetched directly from the backend, so we can ignore this file.

Reply:
I know, but if the file can be ignored, it should be deleted, and if we are keeping it, we need to be consistent with what the backend returns.

I prefer to keep it because we will use it for future tests.

Reply:
Cool! I fix it.

Reply:
Hello Ashikka! 

I already did, since I have some time for coding now, I will be merging all the PRs that have been created in the past days :)

Sorry for bothering you in weekend!